marklogic-data-hub/src/main/java/com/marklogic/hub/util/json/JSONSerializer.java

```diff
@@ -0,0 +1,26 @@
+package com.marklogic.hub.util.json;
+
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.io.OutputStream;
+
+public class JSONSerializer {
+    private static Logger logger = LoggerFactory.getLogger(JSONSerializer.class);
+
+    final OutputStream out;
+
+    public JSONSerializer(OutputStream out) {
+        this.out = out;
+    }
+
+    public void serialize(Object val) throws IOException {
```

Comment:
Can be renamed to `public void write(Object val)`

Reply:
Yes, match with JSONStreamWriter class.